checkstyle/spotbugs-exclude.xml

```diff
@@ -44,6 +47,7 @@
             <Bug category="PERFORMANCE" />
             <Bug pattern="URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD" />
             <Bug pattern="DMI_RANDOM_USED_ONLY_ONCE" />
+            <Bug pattern="FE_FLOATING_POINT_EQUALITY" />
```

Comment:
I'm not sure we want this one. It sounds like a way to get flaky tests. Perhaps a utility function in Asserts would help?

Reply:
Okay, I will search where spotbugs was pointing this issue and replace with a new function in AssertExtensions.
Junit already has special floating point equality assertions, I can't remember why I decided to suppress the warning and not to use junit builtin support. I will be back with a better answer

Reply:
@tkaitchuck  done, I have added a new method to AssertExtensions and removed the exclusion